checkstyle/import-control.xml

```diff
@@ -110,6 +110,7 @@
     <subpackage name="requests">
       <allow pkg="org.apache.kafka.common.protocol" />
       <allow pkg="org.apache.kafka.common.network" />
+      <allow pkg="org.apache.kafka.common.requests" />
```

Comment:
yes.  otherwise, it complains about attempts to import inner classes from the same package.